compiler/src/iree/compiler/Codegen/LLVMGPU/Passes.cpp

```diff
@@ -636,145 +636,6 @@ void addGPUWinogradVectorizePassPipeline(OpPassManager &funcPassManager) {
   funcPassManager.addPass(createOptimizeTensorInsertExtractSlicesPass());
 }
 
-//===---------------------------------------------------------------------===//
```

Comment:
Did you do a sweep for any dead passes? Fine if not but would be good to do as a follow up then.